PublicLands.tm2/style.mss

```diff
@@ -1,25 +1,130 @@
 Map { font-directory: url("./fonts"); }
 
-#lands[adminLevel='federal'] {
-    polygon-fill:red;
-}
-#lands[adminLevel='state'] {
-    polygon-fill:green;
-}
-#lands[adminLevel='county'] {
-    polygon-fill:yellow;
-}
-#lands[adminLevel='local'] {
-    polygon-fill:blue;
-}
-
 #lands {
-    text-face-name: 'Source Sans Pro Regular';
-    text-name:'[adminAgency]';
+	text-face-name: 'Noto Sans Regular';
+    text-name:'[loc_nm]';
     text-size: 12;
-    text-placement:point;
+    text-placement:line;
     text-fill:black;
-    text-min-distance:100;
     text-min-padding:1;
-    text-wrap-width:100;
-}
\ No newline at end of file
+   	text-wrap-width:100;
+	polygon-opacity:1;
+	[zoom >= 14] {
+    	line-width:1;
+    	line-color:black;
+	}
+	[adminLevel='state'] {
+    	polygon-fill:#f781bf;
+	}
+	[adminLevel='county'] {
+    	polygon-fill:#ffff33;
+	}
+	[adminLevel='local'] {
+    	polygon-fill:#377eb8;
+	}
+	[adminAgency='Bureau of Land Management'][type!='wilderness area'] {
+		polygon-fill: #e41a1c;
+	}
+	[adminAgency='Bureau of Reclamation'][type!='wilderness area'] {
+		polygon-fill: #984ea3;
+	}
+	[adminAgency='Forest Service'][type!='wilderness area'] {
+		polygon-fill: #4daf4a;
+	}
+	[adminAgency='U.S. Forest Service'][type!='wilderness area'] {
+        polygon-fill: #4daf4a;
+    }
+	[adminAgency='National Park Service'][type!='wilderness area'] {
+		polygon-fill: #a65628;
+	}
+	[adminAgency='Department of Defense']{
```

Comment:
Instead of setting the same style a bunch of times in different blocks, you can do:
```
[adminAgency='Department of Defense'],[adminAgency='Army Corps of Engineers'],[adminAgency='Air Force'] {
         polygon-fill: #ff7f00;
 }
```